ui/app/templates/components/topo-viz.hbs

```diff
@@ -0,0 +1,24 @@
+<div data-test-topo-viz class="topo-viz {{if this.isSingleColumn "is-single-column"}}" {{did-insert this.buildTopology}} {{did-insert this.captureElement}}>
+  <FlexMasonry
+    @columns={{if this.isSingleColumn 1 2}}
+    @items={{this.topology.datacenters}}
+    @withSpacing={{true}} as |dc|>
+    <TopoViz::Datacenter
+        @datacenter={{dc}}
+        @isSingleColumn={{this.datacenterIsSingleColumn}}
+        @isDense={{this.isDense}}
+        @heightScale={{this.topology.heightScale}}
+        @onAllocationSelect={{this.associateAllocations}}
+        @onNodeSelect={{this.showNodeDetails}} />
+  </FlexMasonry>
+
+  {{#if this.activeAllocation}}
+    <svg data-test-allocation-associations class="chart topo-viz-edges" {{window-resize this.computedActiveEdges}}>
+      <g transform="translate({{this.edgeOffset.x}},{{this.edgeOffset.y}})">
+        {{#each this.activeEdges as |edge|}}
+          <path data-test-allocation-association class="edge" d={{edge}} />
+        {{/each}}
+      </g>
+    </svg>
+  {{/if}}
```

Comment:
The resizing might be a bug, but the lines pointing to weird places is a quirk of the fixtures.

I didn't bother making sure allocations weren't "placed" on clients that didn't have room for them. The result is allocations taking up more than 100% of the width. So that line going to the empty client is actually going to the memory rect for the client in the left column, except it's beyond the width of the client.

It would be nice to make this better at some point, but I'm not sure how important it is since it's just for our local dev experience.

Reply:
ah understood, doesn’t seem worth fixing to me then, thanks